perf/benchmark/setup_test.sh

```diff
@@ -54,6 +55,7 @@ function run_test() {
       --set server.inject="${ISTIO_INJECT}"  \
       --set client.injectL="${LINKERD_INJECT}" \
       --set server.injectL="${LINKERD_INJECT}" \
+      --set values.galley.enableAnalysis="${GALLEY_ANALYSIS}" \
```

Comment:
This call to helm is for the templates not the istio install?

Reply:
oh, yeah, sorry, need to rename this PR